app/views/stream/_about.erb

```diff
@@ -0,0 +1,25 @@
+<div class="container about">
+  <section class="page-header">
+    <h1>Activity streams</h1>
+  </section>
+
+  <section class="main">
+    <article class="article-block">
+      <div class="article-contents">
+        <h4 class="header">How it works</h4>
+        <p> Activity Streams show you how other people approach to the problems. </p>
+        <p> When you have submitted a solution to at least one exercise, then the stream starts getting filled with the solutions that other people have submitted to the same exercise.</p>
+        <p> It will differentiate your submissions from the others by <a href="http://exercism.io/languages">langauge track</a> </p>
+        <p> You will be able to view and comment on others' submissions <p>
+      </div>
+    </article>
+    <hr>
+    <article class="article-block">
+      <div class="article-contents">
+        <h4 class="header">Track Mentor</h4>
+        <p>Want to be a Track Mentor? Working on exercises and giving great feedback is the best way to get noticed!</p>
```

Comment:
Contributing reviews should be something that everybody does, so the 'specialness' of it should not be emphasised. I'd leave anything about being a track mentor off.
> Once you've submitted a solution, you should review others' solutions, ask questions about why they've done things the way they have, or note different approaches to solving the problems.

(or something.)